app/domain/conjur/fetch_trusted_proxies.rb

```diff
@@ -0,0 +1,81 @@
+# frozen_string_literal: true
+
+require 'command_class'
+
+module Conjur
+
+  FetchTrustedProxies ||= CommandClass.new(
+    dependencies: {
+        role_cls:                 ::Role,
+        validate_account_exists:  ::Authentication::Security::ValidateAccountExists.new,
+        logger:                   Rails.logger
+    },
+    inputs: %i(account)
+  ) do
+
+    def call
+      validate_account_exists
+      fetch_trusted_proxies_list
+      delete_duplications_in_list
+      trusted_proxies_list
+    rescue => e
+      raise Errors::Conjur::TrustedProxiesMissing.new(
+          host_id,
+          e.inspect
+      )
+    end
+
+    private
+
+    def validate_account_exists
+      @validate_account_exists.(
+          account: @account
+      )
+    end
+
+    def trusted_proxies_list
+      @proxies_list
+    end
+
+    def delete_duplications_in_list
+      @logger.debug(LogMessages::Conjur::DeletingTrustedProxiesDuplications.new)
+      if @proxies_list
+        @proxies_list = @proxies_list.uniq { |cidr| [cidr.to_s]}
+      end
+      @logger.debug(LogMessages::Conjur::TrustedProxiesAmount.new(@proxies_list.length))
+    end
+
+    def fetch_trusted_proxies_list
```

Comment:
This is a fetch from policy configuration, right ?
Are you assuming that policy configuration of trusted-proxies will contain duplications ?
If yes - just notice that it is not a best practice for configuring entities. The best practice is to disallow duplication when user tries to enter it

Reply:
Please take it with @InbalZilberman to prioritise it and understand the risks of such change in load policy